applications/nrf_desktop/CMakeLists.txt

```diff
@@ -61,3 +61,5 @@ endif()
 if(CONFIG_BT_RPMSG_NRF53)
   assert_exists(hci_rpmsg_CONF_FILE)
 endif()
+
+zephyr_linker_sources(SECTIONS nrf_desktop.ld)
```

Comment:
should be included only when config channel is enabled.

Reply:
btw. Shouldn't the commit be tagged as `applications: nrf_desktop:` ? Since it only affect this application.